.gitignore

```diff
@@ -28,6 +28,7 @@ dist/
 lib/
 transpiled/
 app/scripts/modules/core/src/styleguide/src/public/
+coverage
```

Comment:
what's this guy?

Reply:
Somehow I ended with code coverage, no idea where it came from?